interfaces/cython/cantera/yaml2ck.py

```diff
@@ -287,7 +287,7 @@ def build_thermodynamics_text(
         if spec.thermo.max_temp > max_temp:
             max_temp = spec.thermo.max_temp
 
-        note = input_data["thermo"].get("note", "")
+        note = f"{input_data['thermo'].get('note')}"
```

Comment:
```suggestion
        note = str(input_data['thermo'].get('note', ''))
```
Also, the default value of an empty string.

Reply:
Np. The f-string did interpret `None` correctly, but I don't have an issue with handling this explicitly using the `str` conversion.